### PR TITLE
[Feat] Add pr labeler and pr template with some project description to readme.

### PR DESCRIPTION
**Pull request by moojing**
## Summary

- add pr labeler and pr template.
- add description of project structure to readme.

## Related Links

**Asana**:  - 

**Mockup**:   - 
## Checklist

- [x] Pasted Asana or Mockup link.
- [x] Tagged labels.

## Screenshot/Gif
Screenshot 2022-12-21 at 12 46 23 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
-# Getting Started with Create React App
+# Project Description
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This project is a new version of the blocto wallet for the web user to interact with dApps.
```
Comment (mordochi) [inline README.md:3]: Suppose we can add the description to specify that this repo only contains the frontend part.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+
+See old project repo [here](https://github.com/portto/wallet-webapp).
+
+And in this new we use x-state to do the states management. See the document [here](https://xstate.js.org/docs/guides/states.html#api).
```
Comment (mordochi) [inline README.md:7]: > And in this new `repo` we use x-state to do the states management

File: README.md [hunk 3]
```diff
@@ -1,6 +1,22 @@
...
+
+## Project Structure
+
+There're couple of folders under `src`:
```
Comment (mordochi) [inline README.md:11]: `a couple of`

**Review by mordochi — APPROVED** [review]
Just minor changes to be made. Thank you for handling this!

**Comment by moojing** [comment]
great, thanks for the useful advice.

**Pull request merged**